library/ajax/turnoff_birthday_alert.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Turn off Birthday alert .
```

Comment:
for me:
change to 'Turn off/on birthday alert'
(note that if uncheck the checkbox after check it, it will turn it back on, which is why will change this comment)